url-whitelist-policy/src/main/java/io/apiman/plugins/urlwhitelist/UrlWhitelistPolicy.java

```diff
@@ -23,13 +24,13 @@
  * @author Pete Cornish {@literal <outofcoffee@gmail.com>}
  */
 public class UrlWhitelistPolicy extends AbstractMappedPolicy<UrlWhitelistBean> {
+    private static final Messages MESSAGES = new Messages("io.apiman.plugins.urlwhitelist", "UrlWhitelistPolicy");
     private static final String APIMAN_GATEWAY = "/apiman-gateway";
-    private static final Messages MESSAGES = Messages.getMessageBundle(UrlWhitelistPolicy.class);
 
     /**
      * Cache of precompiled URL patterns. Note that {@link Pattern} is thread-safe.
      */
-    private HashMap<String, Pattern> patternMap;
+    private static final Map<String, Pattern> PATTERN_MAP = new ConcurrentHashMap<>();
```

Comment:
This is a good change.

However, it's worth pointing out (for future reference) that every policy class is automatically a singleton.  So in this particular case it doesn't matter whether it's static or not.  Even so, I think the intent is more clear now. :)